datameta/views/default.py

```diff
@@ -26,12 +26,20 @@
 
 @subscriber(BeforeRender)
 def add_global(event):
```

Comment:
Maybe it is better to not explicitly add the appsettings and use an iterable here?

Reply:
Or encapsulate in a small helper function